src/components/studies/app-design/AppDesign.tsx

```diff
@@ -590,38 +886,73 @@ const AppDesign: React.FunctionComponent<
                     rows={8}
                     rowsMax={10}
                     placeholder="Lorem ipsum"
-                    inputProps={{ style: { width: '100%' } }}
+                    inputProps={{ style: { width: '100%' }, maxLength: 500 }}
                   />
                 </FormControl>
               </FormGroup>
             </Subsection>
             <Subsection heading="Information about the Study Leads">
               <FormGroup className={classes.formFields}>
                 <FormControl className={classes.firstFormElement}>
-                  <SimpleTextLabel htmlFor="lead-investigator-input">
-                    Lead pricinple investigator
-                  </SimpleTextLabel>
-                  <SimpleTextInput
-                    className={classes.informationRowStyle}
-                    id="lead-investigator-input"
-                    placeholder="First and Last name"
-                    value={appDesignProperties.leadPrincipleInvestigator}
-                    onChange={e => {
-                      setAppDesignProperties({
-                        ...appDesignProperties,
-                        leadPrincipleInvestigator: e.target.value,
-                      })
-                    }}
-                    onBlur={() => updateAppDesignInfo()}
-                    multiline
-                    rows={1}
-                    rowsMax={1}
-                    inputProps={{ style: { fontSize: '15px', width: '100%' } }}
-                  />
+                  <Box marginLeft="8px">Lead Principle Investigator*</Box>
+                  <div>
+                    <button
```

Comment:
Let's pull out the principle investigator to be an independent control since we'll need to reuse it in launch study.
The logic  of getting the lead researchers `getLeadResearchAccount` should live there. This control should take token, orgMembership, and selectedValue as props and have an onChange() callback that returns selected user